packages/devextreme/testing/tests/DevExpress.ui.widgets/list.markup.tests.js

```diff
@@ -278,6 +278,9 @@ if(devices.real().deviceType === 'desktop') {
                         items: [{ text: 'Item_1' }, { text: 'Item_2' }, { text: 'Item_3' }],
                     }, options))
             });
+            const localizedRoleDescription = 'list';
```

Comment:
If you trying to check that custom value can be loaded to the dictionary, let's use move explicit custom value
```suggestion
            const localizedRoleDescription = 'My Custom List';
```

Reply:
fixed